src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -85,7 +121,28 @@ const AnnotationActivity = ({
                     <div>
                         <ActivityTimestamp date={createdAtTimestamp} />
                     </div>
-                    <ActivityMessage id={id} tagged_message={message} getUserProfileUrl={getUserProfileUrl} />
+                    {isEditing ? (
+                        <CommentForm
+                            isDisabled={isDisabled}
+                            className={classNames('bcs-Comment-editor', {
```

Comment:
Should we be re-using this classname?

Reply:
No, I've added a new selector that is specific to AnnotationActivity. The styling is also slightly different (border-top: none).